python/ray/rllib/agents/impala/impala.py

```diff
@@ -116,6 +119,10 @@ def _init(self, config, env_creator):
         self.local_evaluator = self.make_local_evaluator(
             self.env_creator, policy_cls)
 
+        graph = self.local_evaluator.policy_map[DEFAULT_POLICY_ID].sess.graph
```

Comment:
Consider adding this to ray upstream

Reply:
I talked to @nikola-j, we agreed we'll do this once in a while (to send these small changes along with bigger ones).